.api/v1.5.0/schema.pbiviz.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
So I considered this - however I saw that users who don't have powerbi-visuals-tools up-to-date will fail this step. This is why I considered this option instead. thoughts?